### PR TITLE
Created Authorization page

### PR DESCRIPTION
**Pull request by PeterPavlatos**
Created Authorization page

image

### CODE REVIEW

File: client/pages/form-pages/authorize.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <p className='font-bold'>
+              We certify that the information contained in this application, is to the best of our
+              knowledge correct, complete and has been submitted with the concurrence of our Council
+              or Board.
+            </p>
+
+            <Field
+              name='projectManagerSignature'
+              type='text'
+              label='Project Manager, Name and Signature'
```
Comment (karthikktamilmani-freshworks) [inline client/pages/form-pages/authorize.tsx:32]: Signature should be a separate field. Use something like this for signature https://www.npmjs.com/package/react-signature-canvas

File: client/pages/form-pages/authorize.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+              maxLength={300}
+            ></Field>
+            <Field
+              name='financialOfficerSignature'
+              type='text'
+              label='Financial Officer, Name and Signature'
```
Comment (karthikktamilmani-freshworks) [inline client/pages/form-pages/authorize.tsx:38]: Similarly here!

File: client/pages/form-pages/authorize.tsx (hunk 1 above)
Comment (PeterPavlatos) [reply client/pages/form-pages/authorize.tsx:83]: Added signature-canvas package. Looks great , thanks.
image

File: client/pages/form-pages/authorize.tsx (hunk 2 above)
Comment (PeterPavlatos) [reply client/pages/form-pages/authorize.tsx:91]: Same here.

File: client/components/Button.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ export interface ButtonProps {
   type?: 'submit' | 'reset' | 'button';
   disabled?: boolean;
   children: React.ReactNode;
+  classes?: string;
```
Comment (arranfw) [inline client/components/Button.tsx:11]: This leaves us open to having divergent button styles across the app and circumvents the component's consistency 

It doesn't look like this prop is used in this PR, can we remove this?

File: client/pages/form-pages/authorize.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+              or Board.
+            </p>
+
+            <Field
+              name='projectManagerSignature'
+              type='text'
+              label='Project Manager, Name and Signature'
+              maxLength={300}
+            ></Field>
+            <SignPad />
```
Comment (arranfw) [inline client/pages/form-pages/authorize.tsx:86]: Have we looking into the accessibility implications of this field? Is it a required field? I can't imagine a keyboard or SR user would be able to fill this out very easily

File: client/pages/form-pages/authorize.tsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  return (
+    <Formik initialValues={initialValues} onSubmit={() => {}}>
+      {({ values }: FormikProps<AuthorizationInterface>) => (
+        <Form className='flex justify-center'>
+          <div className='w-2/4 p-4 gap-y-6 bg-white flex flex-col items-center'>
+            <div className='mb-4 flex items-center flex-col'>
+              <h1 className='text-xl font-medium text-bcBluePrimary'>
+                {FormStepTitles[FormSteps.AUTHORIZATION]}
+              </h1>
+              <h3>Please proivde detailed information for your application.</h3>
```
Comment (arranfw) [inline client/pages/form-pages/authorize.tsx:71]: These headings aren't sequential, this is probably more appropriate as a `<p>` as it doesn't represent a new/nested section of content

reference: https://www.w3.org/WAI/tutorials/page-structure/headings/

File: client/components/Button.tsx (hunk 3 above)
Comment (PeterPavlatos) [reply client/components/Button.tsx:11]: Removed 'classes'.

File: client/pages/form-pages/authorize.tsx (hunk 4 above)
Comment (PeterPavlatos) [reply client/pages/form-pages/authorize.tsx:86]: No we have looked into it. Was not part of original requirement but would be good to have.

File: client/pages/form-pages/authorize.tsx (hunk 5 above)
Comment (PeterPavlatos) [reply client/pages/form-pages/authorize.tsx]: Agreed. Changed to p tag

**Review by karthikktamilmani-freshworks — APPROVED** [review]

**Pull request merged**